assets/training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -28,6 +28,15 @@ inputs:
     description: Huggingface model id (https://huggingface.co/<model_id>). A required parameter for Huggingface model framework. Can be provided as input here or in model_download_metadata JSON file.
     optional: true
 
+  model_flavor:
```

Comment:
can we handle it internally? We can make OSS default and if it fails use hftransformers as we've discussed previously?